packages/compass-components/src/components/icon.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
Also kinda related to my other comment: I think it's good that we want to just re-export some leafygreen components here, will make maintaining versions across Compass easier, but I can't see a benefit in testing them. Do you think we get something from testing components that we just re-export?

Reply:
Sounds good, I think having one test doesn't hurt but I can't really think of a good reason to have it hah. I'll leave this one since we've got one, but future ones I'll just re-export.